### PR TITLE
Added: RST-328, staff-only case summary by language

### PR DESCRIPTION
**Pull request by richerry**
Court staff would like to view a summary of cases by language.

### CODE REVIEW

**Comment by AlexaVB** [comment]
Looks good @richerry 
Please review @robyoung 
At this stage I won't write Cucumber tests for this.

File: make_a_plea/middleware.py [hunk 1]
```diff
@@ -34,8 +39,22 @@ def process_response(self, request, response):
...
+
+    def process_exception(self, request, e):
+        if isinstance(e, BadRequestException):
+            return HttpResponse(
+                json.dumps(
+                    {"error": e.message},
+                    indent=4),
+                status=400,
+                content_type="application/json",
+            )
```
Comment (robyoung) [inline make_a_plea/middleware.py]: Won't this respond with a JSON error for all 400s including those on the frontend?

Comment (richerry) [inline make_a_plea/middleware.py]: No, I created the BadRequestException class and only used it where I want this behaviour.

Comment (robyoung) [inline make_a_plea/middleware.py]: I'm not sure the name is really clear enough then. I wouldn't automatically assume that the `BadRequestException` is just used for returning JSON. Would using JSON in the exception name make it clearer?

Comment (richerry) [inline make_a_plea/middleware.py]: It's not JSON specific. It's for any bad request we want to handle consistently. One could imagine the middleware doing the content type detection based on the incoming accept headers. So it's not about the response being JSON, it's about using exception middleware.

**Comment by robyoung** [comment]
@richerry I've done the rebase and the tests are passing.

**Comment by richerry** [comment]
Tested with fixtures generated in another ticket. All good.

**Pull request merged**